src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -118,6 +119,7 @@ public TypeWithAnnotations GetInferredReturnType(ConversionsBase conversions, Nu
                 var compilation = Binder.Compilation;
                 NullableWalker.Analyze(compilation,
                                        lambda: this,
+                                       (Conversions)conversions,
```

Comment:
>(Conversions) [](start = 39, length = 13)

Can the method argument type be changed to `Conversions`?

Reply:
No. `NullableWalker` needs a non-null `Conversions` in order to correctly analyze code.

---
In reply to: [269162419](https://github.com/dotnet/roslyn/pull/34429#discussion_r269162419) [](ancestors = 269162419)

Reply:
As we discussed offline, the question was whether the argument to this method could be declared `Conversions` rather than `ConversionsBase`.

---
In reply to: [269763362](https://github.com/dotnet/roslyn/pull/34429#discussion_r269763362) [](ancestors = 269763362,269162419)